include/ck/utility/f8_utils.hpp

```diff
@@ -0,0 +1,251 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2023, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+namespace ck {
+
+using f8_t   = uint8_t;
+using half_t = _Float16;
```

Comment:
Oh I've overlooked this. Please move the fp8 type alias to `data_type.hpp` and remove the `half_t` type alias. As I said for such simple type aliases definitions I think we should keep all the data type definitions in `data_type.hpp` header. I imagine an exception to this rule only when we would define our own class for a specific data type.

Reply:
Fixed in [2ee1c0a](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/708/commits/2ee1c0a70a305b00770c51a6ad585895992b327c)